xep-0068.xml

```diff
@@ -115,7 +121,7 @@
 
 <section1 topic='Use Cases' anchor='usecases'>
     <section2 topic='Unspecified Form' anchor='usecases-unspecified'>
-      <p>These are forms that do not have a hidden field of name FORM_TYPE.
+      <p>These are forms with type=&quot;form&quot; or type=&quot;result&quot; that do not have a hidden field of name FORM_TYPE or forms with type=&quot;submit&quot; that do not have any field of name FORM_TYPE.
```

Comment:
nit: you don't need to use &quot; here and in every place you introduced it with this change.

Reply:
I actually do know, but did nonetheless for consistency with how it was before and because I wasn't sure if any tooling might have issues without.

Reply:
If anything, tooling is more unhappy *with* `&quot;`. As a general rule of thumb, anything which is valid XML goes, though :).